gui/wxpython/main_window/frame.py

```diff
@@ -309,6 +309,10 @@ def _createMapNotebook(self):
             agwStyle=notebook_style,
         )
         self.mapnotebook.SetArtProvider(aui.AuiDefaultTabArt())
+        # bindings
+        self.mapnotebook.Bind(
+            aui.EVT_AUINOTEBOOK_PAGE_CHANGED, self.OnMapNotebookPageChanged
```

Comment:
There is _onMapDisplayFocus in lmgr/frame.py, which seems to be doing what you need, why this new method?